src/components/ProjectBillingPage/ProjectBillingPage.module.css

```diff
@@ -0,0 +1,157 @@
+.Page {
+    margin: 0;
+    background: #f1f1f1;
+    max-height: 100vh;
+    display: grid;
+    grid-template-rows: auto 1fr auto;
+    /* overflow: hidden; */
+  }
+  .TopBarSection {
+    background-color: black;
+  }
+  .MainSection {
+    /* divided into two columns */
+    background-color: #f1f1f1;
+    display: grid;
+    grid-template-columns: 1.3fr 6fr;
+    overflow: hidden;
+  }
+  .SideBarSection {
+    overflow: hidden;
+    overflow-y: scroll;
+    height: calc(100vh - 5rem);
+  }
+  
+  .SideBarSection::-webkit-scrollbar {
+    display: none;
+    min-width: fit-content;
+  }
+  
+  .SideBarSection {
+    -ms-overflow-style: none;
+    scrollbar-width: none;
+  }
+  
+  .MainContentSection {
+    /* divided into two rows */
+    display: grid;
+    grid-template-rows: 80px 4fr;
+    overflow: auto;
+  }
+
+  .SmallContainer{
```

Comment:
Have u ever tried, padding: 0 2rem 2rem;

Reply:
noted and changed